Source/CombatExtended/CombatExtended/Comps/HediffComp_Venom.cs

```diff
@@ -14,7 +14,7 @@ public override void CompPostPostAdd(DamageInfo? dinfo)
         {
             base.CompPostPostAdd(dinfo);
 
-            _venomPerTick = Props.VenomPerSeverity * parent.Severity / parent.pawn.BodySize * parent.pawn.GetStatValue(StatDefOf.ToxicSensitivity);
+            _venomPerTick = Props.VenomPerSeverity * parent.Severity / parent.pawn.BodySize * parent.pawn.GetStatValue(StatDefOf.ToxicResistance);
```

Comment:
From the name, I would guess that `ToxicSensitivity` is 0 when immune and 1 normally.  Toxic resistance would be 1 when immune and 0 normally.  In that case, this needs to be `* (1 - ...`